### PR TITLE
code reformat (new)

### PR DESCRIPTION
**Pull request by beidouz**
## Notice

It is not allowed to submit your PR to the master branch directly, please submit your PR to the master-pre-merge branch.

## Description

Please include a brief summary of the change that this pull request proposes. Include any relevant motivation and context. List any dependencies required for this change.

- Reformat code in all modules
- update license headers in all modules for consistency

Fixes Issue # .

## Type of change

Insert **x** into the following checkboxes to confirm (eg. [x]):
- [ ] Bug fix.
- [ ] New feature.
- [x ] Enhancement.
- [ ] Unit test.
- [ ] Breaking change (a fix or feature that causes existing functionality to not work as expected).
- [ ] Requires documentation update.

## Testing

Please describe the tests you used to validate this pull request. Provide any relevant details for test configurations as well as any instructions to reproduce these results.

-

## Verification

Insert **x** into the following checkboxes to confirm (eg. [x]):
- [ ] I have self-reviewed my own code and conformed to the style guidelines of this project.
- [ ] New and existing tests pass locally with my changes.
- [ ] I have added tests for my fix or feature.
- [ ] I have made appropriate changes to the corresponding documentation.
- [ ] My code generates no new warnings.
- [ ] Any dependent changes have been made.

### CODE REVIEW

File: modCrypto/test/org/aion/crypto/ecdsa/ECDSATest.java [hunk 1]
```diff
@@ -1,30 +1,59 @@
...
+ *     along with the aion network project source files.
+ *     If not, see <https://www.gnu.org/licenses/>.
  *
- * <p>The aion network project leverages useful source code from other open source projects. We
- * greatly appreciate the effort that was invested in these projects and we thank the individual
- * contributors for their work. For provenance information and contributors please see
- * <https://github.com/aionnetwork/aion/wiki/Contributors>.
+ * Contributors:
+ *     Aion foundation.
+ */
```
Comment (AlexandraRoatis) [inline modCrypto/test/org/aion/crypto/ecdsa/ECDSATest.java]: there are two licenses here

File: modMcf/src/org/aion/mcf/config/CfgPrune.java [hunk 2]
```diff
@@ -25,7 +25,13 @@
...
  * Contributors to the aion source files in decreasing order of code volume:
  *     Aion foundation.
- ******************************************************************************/
+ *     <ether.camp> team through the ethereumJ library.
+ *     Ether.Camp Inc. (US) team through Ethereum Harmony.
+ *     John Tromp through the Equihash solver.
+ *     Samuel Neves through the BLAKE2 implementation.
+ *     Zcash project team.
+ *     Bitcoinj team.
+ */
```
Comment (AlexandraRoatis) [inline modMcf/src/org/aion/mcf/config/CfgPrune.java]: This class should have the short license version.

Comment (beidouz) [reply modMcf/src/org/aion/mcf/config/CfgPrune.java]: nice catch, ty!

File: modMcf/src/org/aion/mcf/trie/TrieImpl.java [hunk 3]
```diff
@@ -17,15 +17,10 @@
...
- *     open source projects. We greatly appreciate the effort that was
- *     invested in these projects and we thank the individual contributors
- *     for their work. For provenance information and contributors
- *     please see <https://github.com/aionnetwork/aion/wiki/Contributors>.
- *
- * Contributors to the aion source files in decreasing order of code volume:
+ * Contributors:
  *     Aion foundation.
- *******************************************************************************/
+ */
```
[the remark this answers is not in the document]
Comment (AionJayT) [reply modMcf/src/org/aion/mcf/trie/TrieImpl.java]: Agree with @AlexandraRoatis

**Pull request merged**